composer.json

```diff
@@ -29,7 +29,8 @@
     },
     "require-dev": {
 
-        "phing/phing": "2.14.0"
+        "phing/phing": "2.14.0",
+        "phpunit/phpunit": "^5.7"
```

Comment:
Is phpunit needed in the actual codebase?
(or is it more of a developer/testing tool?)